adapters/v1/syft_test.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/kinbiko/jsonassert"
+	"github.com/kubescape/kubevuln/core/domain"
+	"gotest.tools/v3/assert"
+)
+
+func fileContent(path string) []byte {
+	b, _ := os.ReadFile(path)
+	return b
+}
+
+func Test_syftAdapter_CreateSBOM(t *testing.T) {
+	tests := []struct {
+		name    string
+		imageID string
+		format  string
+		options domain.RegistryOptions
+		wantErr bool
+	}{
+		{
+			name:    "hello-world",
+			imageID: "library/hello-world@sha256:aa0cc8055b82dc2509bed2e19b275c8f463506616377219d9642221ab53cf9fe",
+			format:  string(fileContent("testdata/hello-world-sbom.format.json")),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			s := NewSyftAdapter()
```

Comment:
question: is there a reason that the Grype adapter returned an error, while this one does not?

I believe they both solve a similar problem. With Grype, something can go wrong, but with Syft, it can’t?

Reply:
Yes, for the moment it only initializes the struct, which cannot fail... maybe for future proofing I should add error handling in the case this "constructor" needs it in the future?
Let's KISS for the moment.